pkgs/tools/text/angle-grinder/default.nix

```
@@ -16,6 +17,10 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-+l0+zaZSPOk4gJLHZ9LFFbYlZ5vkS68Jg2dWPHSkzKw=";
 
+  passthru = {
+    updateScript = gitUpdater { rev-prefix = "v"; };
```

The gitUpdater can not update `cargoHash`. Use the following instead:

```suggestion
    updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
```